t/02_request/14_uploads.t

```diff
@@ -182,5 +190,33 @@ for (qw/a b c/){
 }
 
 $resp =  dancer_response( 'POST', '/uploads', {files => $files});
-is length($resp->content), 3;
-like $resp->content, qr/a/;
+is $resp->content, 
+    join(';',"a:0cc175b9c0f1b6a831c399e269772661",
+    "b:92eb5ffee6ae2fec3ad71c777531578f",
+    "c:4a8a08f09d37b73795649038408b5f33"),
+    "Expected response for multi uploads";
+
+
+
+## test #23
+# create a file, 256MB of zeros
+$dest_file = File::Spec->catfile($dest_dir, "zeros");
+system("dd if=/dev/zero of=$dest_file count=250000 bs=1024");
```

Comment:
Oh, indeed - hence my comments that I'm not sure if it should be made an author test, or possibly run only if you set an env var or something, or just binned altogether - it's served its purpose, and leaving it could cause unexpected failures if run on low-RAM systems.  BTW, the test file should get automatically cleared up by File::Temp when it removes that temp dir - assuming the test script doesn't crash before that happens of course - so if it OOMs it's probably left behind.